src/wallet/wallet.cpp

```diff
@@ -4145,3 +4148,15 @@ bool CMerkleTx::AcceptToMemoryPool(const CAmount& nAbsurdFee, CValidationState&
 {
     return ::AcceptToMemoryPool(mempool, state, tx, true, NULL, NULL, false, nAbsurdFee);
 }
+
+bool CalculateEstimateType(FeeEstimateMode mode, bool opt_in_rbf) {
```

Comment:
I think it makes more sense to let it be generic and ideally estimatesmartfee would take a FeeEstimateMode and this functions purpose would be to take any requested mode and other wallet parameters and use wallet logic to determine a final mode.  Rather than make all those changes now while not yet needed though, I think I'll just change the RPC interface to conservative estimates to take a string as per https://github.com/bitcoin/bitcoin/pull/10589#pullrequestreview-45713269